ParisBNF/abb/BNFabb94.xml

```diff
@@ -0,0 +1,324 @@
+    </objectDesc>
+    
+    
+  <handDesc>
+     <handNote xml:id="h1" script="Ethiopic">
+        <seg type="script">15th century.</seg>
+        <seg type="ink">Black</seg>
+      </handNote>
+  </handDesc>
+
+</physDesc>
+
+
+                          </msPart>
+
+            <msPart xml:id="p2">
+              <msIdentifier></msIdentifier>
+              <msContents>
+
+                              <msItem xml:id="p2_i1">
+                                   <locus from="34r" to="41v"/>
+                                    <title type="complete" xml:lang="gez" ref="LIT2365Storyo"/>
+                                    <textLang xml:lang="gez"></textLang>
+                                  </msItem>
+
+                                  <msItem xml:id="p2_i2">
+                                       <locus from="41v" to="47r"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT3326Melchiz1"/>
+                                        <textLang xml:lang="gez"></textLang>
+                                      </msItem>
+
+                                      <msItem xml:id="p2_i3">
+                                           <locus from="47r" to="53r"/>
+                                            <title type="complete" xml:lang="gez" ref="LIT3327Melchiz2"/>
+                                            <textLang xml:lang="gez"></textLang>
+                                          </msItem>
+
+                                          <msItem xml:id="p2_i4">
+                                               <locus from="53r" to="55r"/>
+                                                <title type="complete" xml:lang="gez" ref="LIT3332Melchiz"/>
+                                                <textLang xml:lang="gez"></textLang>
+                                              </msItem>
+</msContents>
+
+<physDesc>
+    
+    <objectDesc>
+    <layoutDesc>
+        <layout columns="2" writtenLines="19 21">
+        </layout>
+    </layoutDesc>
+    </objectDesc>
+    
+  <handDesc>
+     <handNote xml:id="h2" script="Ethiopic">
+        <seg type="script">15th century.</seg>
+        <seg type="ink">Black</seg>
+      </handNote>
+  </handDesc>
+
+<additions>
+<list>
+  <item xml:id="a2">
+    <desc type="ScribalSupplication"/>
+    <locus from="41r" to="41v"/>
+    <q xml:lang="gez">ዘንተ፡ መጽሐፈ፡ አጥረዩ፡ አባ፡ <persName role="scribe" ref="PRS14418Nob">ኖብ፡</persName> ወሐራ፡ ክርስቶስ፡ ይኩኖሙ፡ ተዝካረ፡ እስከ፡ ለዓለም፡ ወመርሐ፡ ለመንግሥተ፡ ሰማያት፡ አሜን። ዘንተ፡
+    መጽሐፈ፡ ዘጸሐፎ፡ ወዘአጽሐፎ፡ ወለ፡ ዘአንበቦ፡ ወለዘተርጐሞ፡ ወዘሰምዐ፡ ቃላቲሁ፡ ይምሐሮሙ፡ እግዚአብሔር፡ ወይስረይ፡ ኃጢአቶሙ፡ በጸሎታ፡ ለማርያም፡ እመ፡ ብርሃን።
+  ወበጸሎቶሙ፡ ለሩፋኤል፡ ወመልከጼዴቅ፡ ወይክፍለነ፡ ምቅዋመ፡ በሐዳስ፡ ንድቅ፡ እግዚአብሔር፡ እግዚእ፡ መኰንነ፡ ጽድቅ፡ ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን።</q>
+  </item>
+
+  <item xml:id="a3">
+    <desc type="ScribalSupplication"/>
+    <locus target="#55r"/>
+    <q xml:lang="gez">ዛቲ፡ መጽሐፍ፡ ዘጸሐውክዋ፡ አነ፡ <persName role="scribe" ref="PRS14573HaraKrestos" >ሐራ፡ ክርስቶስ፡</persName> ወዘአጽሐፈኒ፡ አባ፡ ኖብ፡ ከመ፡ ይኩነነ፡ ተዝካረ፡ በጸሎታ፡ ለእግዝእትነ፡ ለእመ፡ ብርሃን፡
```

Comment:
I would also encode Abba Nob with pers id. But reading these two supplications, I wonder whether role="scribe" is correct. I would rather suggest role="patron" or "sponsor". What do you think?